confidant/settings.py

```diff
@@ -606,6 +606,17 @@ def str_env(var_name, default=''):
 ENABLE_SAVE_LAST_DECRYPTION_TIME = bool_env('ENABLE_SAVE_LAST_DECRYPTION_TIME')
 
 # Add any certificate authorities
+# Should be in encrypted settings following this format (where name is
+# the name of the environment) and key ids must be unique:
+# [
```

Comment:
@erickduran just caught this now, I don't think this is the correct format anymore.